packages/web-react/src/components/Box/demo/BoxWithBackgroundGradient.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Box from '../Box';
+
+const BoxWithBackgroundGradient = () => (
+  <>
+    <p>For demo purposes, the boxes have custom padding.</p>
+    <Box padding="space-800" backgroundGradient="primary">
+      Primary Background Gradient
+    </Box>
+    <Box padding="space-800" backgroundGradient="secondary">
+      Secondary Background Gradient
+    </Box>
+    <Box padding="space-800" backgroundGradient={{ mobile: 'primary', tablet: 'secondary', desktop: 'secondary' }}>
```

Comment:
Can we omit the desktop value if it is the same?

```suggestion
    <Box padding="space-800" backgroundGradient={{ mobile: 'primary', tablet: 'secondary' }}>
```

Reply:
Or there is a typo and it should be there:

```suggestion
    <Box padding="space-800" backgroundGradient={{ mobile: 'primary', tablet: 'secondary', desktop: 'primary' }}>
```
?